packages/containers/src/List/utils.js

```diff
@@ -0,0 +1,101 @@
+import cmf from '@talend/react-cmf';
+import get from 'lodash/get';
+import { Map, List } from 'immutable';
+
+function contains(listItem, query, columns) {
+	let item = listItem;
+	if (Map.isMap(listItem)) {
+		item = listItem.toJS();
+	}
+	for (const column of columns) {
+		if (typeof item[column.key] === 'string') {
+			if (item[column.key].toLowerCase().indexOf(query.toLowerCase()) !== -1) {
+				return true;
+			}
+		}
+	}
+	return false;
+}
+
+function getCollection(state, collectionId) {
+	return state.cmf.collections.get(collectionId);
+}
+
+export function getCollectionItems(state, collectionId) {
+	const collection = getCollection(state, collectionId);
+
+	if (Map.isMap(collection)) {
+		return collection.get('items');
+	}
+	return collection;
+}
+
+export function configureGetPagination(state, { collectionId }) {
+	const collection = getCollection(state, collectionId);
+
+	if (Map.isMap(collection)) {
+		return collection.get('pagination');
+	}
+
+	return null;
+}
+
+export function getComponentState(collectionId) {
+	return state => state.cmf.components.getIn(['Container(List)', collectionId || 'default']);
+}
+
+export function sortList(componentState, localConfig, items) {
```

Comment:
Same as filter fn, we need a simple function interface

```javascript
export function sortList(items, sorBy, sortAsc, columnsConfig) {
```
We need also to expect the args are properly given, or can be specifically undefined (with a default value or a specific logic if not defined)